D3D11Engine/D3D11GraphicsEngine.cpp

```diff
@@ -4232,11 +4232,9 @@ void XM_CALLCONV D3D11GraphicsEngine::DrawWorldAround( FXMVECTOR position,
             // Unbind PS
             Context->PSSetShader( nullptr, nullptr, 0 );
         }
-
-        // timer range is between 0 and 100000, for now having bugs with huge values for sin/cos
         VS_ExConstantBuffer_Wind windBuff;
-        windBuff.globalTime = static_cast<float>( Engine::GAPI->GetTotalTimeDW() % 100000 );
-        windBuff.windDir = float3( 0.3f, 0.15f, 0.5f );
+
+        //ApplyWindProps( windBuff ); // Do we need to call wind here too? Why? Or it is just common buff for shader?
```

Comment:
Yes we do, it is required for drawing shadows and shadows shouldn't be static, so if object have animation they should have that animation too.